apps/api/views/profiles.py

```diff
@@ -61,20 +61,24 @@ def create_students_from_csv(request, version):
                         'email': row[4],
                     },
                     'student_id': row[3],
-                    'team': {
-                        'name': row[5],
-                        'klass': request.data.get('klass')
-                    },
                     'klass': request.data.get('klass'),
                 }
```

Comment:
This is for making sure a blank team isn't accidentally created. The team name, `row[5]` must have at least one alphanumeric character to be considered valid.

Reply:
👍